### PR TITLE
Operation leverages AI capabilities

### PR DESCRIPTION
**Pull request by BerSie**
End-users should be make aware that there are actions that use AI functionality. As to my knowledge it's about end-users, I have put the term into the UI vocabulary. If you know scenarios where this information is interesting independent of UI consumption, we might move it to Common.

### CODE REVIEW

File: vocabularies/UI.xml [hunk 1]
```diff
@@ -1549,6 +1549,14 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="IsAIOperation" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Action Function ActionImport FunctionImport">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated operation leverages AI capabilities." />
+        <Annotation Term="Core.LongDescription">
+          <String>This term should end-users make aware that the annotated operation uses AI functionality to process the selected application data.</String>
```
Comment (ralfhandl) [inline vocabularies/UI.xml:1556]:
```suggestion
          <String>This term allows making end-users aware that the annotated operation uses AI functionality to process the selected application data.</String>
```

**Comment by ralfhandl** [comment]
What's the background for this?

**Comment by BerSie** [comment]
> What's the background for this?

AI functions like "Summary" should get a specific design (be it background color, or icon, or ...) to make the end user aware that processing this operation will send the app data to an AI instance.

**Review by ralfhandl — DISMISSED** [review]

File: vocabularies/UI.xml [hunk 2]
```diff
@@ -1549,6 +1549,14 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="IsAIOperation" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Action Function ActionImport FunctionImport">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated operation leverages AI capabilities." />
```
Comment (HeikoTheissen) [inline vocabularies/UI.xml:1554]:
```suggestion
        <Annotation Term="Core.Description" String="The annotated operation is powered by AI" />
```

**Review by HeikoTheissen — COMMENTED** [review]
Suggest rephrasing

File: vocabularies/UI.xml (hunk 2 above)
Comment (BerSie) [reply vocabularies/UI.xml:1554]: sounds more like a marketing campaign, but would be OK for me.

**Review by ralfhandl — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**